### PR TITLE
BUGFIX: FakeQueue should also use the cached message passing

### PR DESCRIPTION
**Pull request by daniellienert**

### CODE REVIEW

**Comment by kdambekalns** [comment]
@daniellienert One tiny thing: Why use a `VariableFrontend` if you serialize yourself? In that case you could use a `StringFrontend`…

**Comment by daniellienert** [comment]
@kdambekalns Sorry I just read about the difference. So I could remove the serialization or switch to the other Frontend - what would be better? Would you define that as breaking?

@dimaip: Sorry I forgot the object configuration and thus I think it could not work before. Are you sure? Would you test it again?

**Comment by kitsunet** [comment]
I would prefer to use VariableFrontend and not do any serialisation in the code.

**Comment by kdambekalns** [comment]
Well, the thing is: the cached content must match "both sides" of the cache. This simply writes to it, but the `JobCommandController` and `Jobmanager` both read and then explicitly use `unserialize()` on the message. So just removing the `serialize()` here will break it.

The content in the cache is a string, since the cache-based message passing was added (oversight during review), so I'd stick to that. But we could switch to a "simpler" cache frontend.

Bottom line: this PR is good, as it matches the rest of the codebase, but I opened #28 as a follow-up.
